syntax = "proto3";

package authcservice;

service MyService {
  rpc GetHelloMessage (HelloRequest) returns (HelloResponse);
}

message HelloRequest {
  string name = 1;
}

message HelloResponse {
  string message = 1;
}


//  ******************************************

service CheckToken{
  rpc GetTokenData(TokenRequest) returns (TokenResponse);
}

message TokenRequest{
  string token= 2;
}

message TokenResponse{
  string message =2;
}
